cmd/validator/validator.go

```diff
@@ -2,7 +2,7 @@
 Validator recusively scans a directory to search for configuration files and
 validates them using the go package for each configuration type.
 
-Currently json, yaml, toml, xml and ini configuration file types are supported.
+Currently json, yaml, toml, xml, ini, and hcl configuration file types are supported.
```

Comment:
Properties files are also supported - we must have missed this function doc for the last PR - please add. Thanks!